### DISCUSSION TITLE
How to remove/hide language from stats?

### DISCUSSION THREAD

**Discussion by rustam-mukhametshin** [codewars/codewars.com#2639] [Q&A]
Hey guys ✋!

Is it possible to remove/hide some language from my story(stats) and its tasks (completed/not) 😑?

For example, couple of years ago, I completed tasks with mysql 😒 , however, right now I work only with Javascript.
So, I would like to somehow remove/hide all related data with mysql from story/stats 😄.

Tried to use google/codewars help etc, nothing 😑.

Thank you for you time :)

**Comment by Blind4Basics** [comment]
Hi,

Unfortunately, you just cannot. You may fill a feature request in the [issue repo](https://github.com/codewars/codewars.com/issues), if you want. That might get support from other users.
